src/modules/Overlay/info.json

```diff
@@ -13,9 +13,12 @@
             "default": 0
         },
         "offset": {
-            "type": "integer",
+            "type": "float",
```

Comment:
type `integer` can also be used as a range.

Reply:
Ohh I didn't knew that, So should I change this to integer?

Reply:
Sure.

Reply:
Done!